core/src/main/java/com/backblaze/b2/json/B2JsonUnionBaseHandler.java

```diff
@@ -118,7 +118,11 @@ protected void initializeImplementation(B2JsonHandlerMap b2JsonHandlerMap) throw
         for (Class<?> subclass : typeNameToClass.values()) {
             for (Field field : B2JsonObjectHandler.getObjectFieldsForJson(subclass)) {
                 final String fieldName = field.getName();
-                final B2JsonTypeHandler handler = b2JsonHandlerMap.getUninitializedHandler(field.getType());
+                final B2JsonTypeHandler handler =
```

Comment:
i can see where this would be better.

...and it makes me wonder why we have two things so similar.
could the contents of the B2JsonObjectHandler version be
moved into the B2JsonHandlerMap version?

Reply:
I'll move the methods to `B2JsonHandlerMap`, and add some comments.